mypy/newsemanal/semanal.py

```diff
@@ -228,6 +211,8 @@ def __init__(self,
             errors: Report analysis errors using this instance
         """
         self.locals = [None]
+        # Saved namespaces from previous passes.
```

Comment:
A more detailed explanation would be nice here. Also, avoid the use of the term 'pass' since this means a different thing in the old semantic analyzer (maybe 'iteration'?).